uikit/views/src/main/java/com/sygic/maps/uikit/views/common/extensions/ContextExtensions.kt

```diff
@@ -78,7 +84,11 @@ fun Context.openUrl(url: String) {
         }
 
         try {
-            startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(parsedUrl)))
+            startActivity(
```

Comment:
nedame pre istotu `fun Context.openUrl(url: String, vararg flags: Int = 0)` aby sa to dalo ovplyvnit, lebo to je extension pramo na zakladny `Context`

Reply:
máš pravdu, supr typ, díky ;) neuvědomil jsem si, že to uvidí všichni ;)